### PR TITLE
Mx4PC Operator 2.5.0 release notes - for May 12 release

### PR DESCRIPTION
**Pull request by zlogic**
This release has relatively few new features:
* Experimental ARM64 support
* Full support of service meshes in an entire namespace (previously, only app/environment pods could use service meshes, and only for incoming client traffic)

### CODE REVIEW

File: content/en/docs/releasenotes/deployment/mendix-for-private-cloud.md [hunk 1]
```diff
@@ -13,6 +13,15 @@ For information on the current status of deployment to Mendix for Private Cloud
...
 ## 2022
 
+### May 12th, 2022
+
+#### Mendix Operator v2.5.0 and Mendix Gateway Agent v2.5.0
+
+* We improved support for injected sidecars - the Operator now works correctly with sidecars injected into any pod, not just app pods. It is now possible to install a service mesh such as [Istio](https://istio.io/) or [Linkerd](https://linkerd.io) for the whole namespace, and route all traffic between pods through the service mesh.
+* We refactored our internal build process - images now share layers as much as possible, reducing download sizes and making it easier to patch CVEs from the base OS layers.
+* A subset of Mendix for Private Cloud components are now available for the ARM64 architecture, allowing you to deploy Mendix apps to machines running Apple Silicon, Raspberry Pi 4, and server-grade ARM64 chips.
+* We have made preparations to support a new, simplified process for obtaining licenses, which will become available soon.
```
Comment (MarkvanMents) [inline content/en/docs/releasenotes/deployment/mendix-for-private-cloud.md]: I'm not sure this is the best place to announce this. It doesn't have any customer impact at present, and is more a marketing statement than a technical one.

I'd like to remove this, unless you are very much attached to it.
```suggestion
```

File: content/en/docs/developerportal/deploy/private-cloud/private-cloud-supported-environments.md [hunk 2]
```diff
@@ -61,10 +61,31 @@ To install the Mendix Operator, the cluster administrator will need permissions
...
+Mendix Operator runs on CPUs with the [x86-64](https://en.wikipedia.org/wiki/X86-64) achitecture.
+
+{{% alert color="info" %}}
+
+Starting with Mendix Operator v2.5.0, container images used in *Connected Mode* also support [ARM64/AArch64](https://en.wikipedia.org/wiki/AArch64). *ARM64* support is experimental at this moment and should only be used for non-production environments.
+
+Only core *Connected mode* features support *ARM64*. The following features **do not** support *ARM64* CPUs at the moment:
+
+* [Migrating to Your Own Registry](/developerportal/deploy/private-cloud-migrating/)
+* CI/CD pipelines in *Standalone mode*
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/private-cloud/private-cloud-supported-environments.md]: We haven't released CI/CD pipelines yet?
```suggestion
```

**Review by MarkvanMents — APPROVED** [review]
Nothing particularly wrong, but can you review the two statements I have commented on and decide if they should be part of this release, or perhaps future ones.

File: content/en/docs/releasenotes/deployment/mendix-for-private-cloud.md (hunk 1 above)
Comment (zlogic) [reply content/en/docs/releasenotes/deployment/mendix-for-private-cloud.md]: Yes, this makes sense.

File: content/en/docs/developerportal/deploy/private-cloud/private-cloud-supported-environments.md (hunk 2 above)
Comment (zlogic) [reply content/en/docs/developerportal/deploy/private-cloud/private-cloud-supported-environments.md]: Yes, indeed - it's best to add this later, when the [Tekton pipelines](https://github.com/mendix/docs/pull/4421) are officially released.

**Comment by MarkvanMents** [comment]
Operator version is now available.

**Pull request merged**